packages/ses/src/tame-function-constructors.js

```diff
@@ -126,11 +126,17 @@ export default function tameFunctionConstructors() {
     '%InertAsyncFunction%',
     '(async function(){})',
   );
-  repairFunction(
-    'AsyncGeneratorFunction',
-    '%InertAsyncGeneratorFunction%',
-    '(async function*(){})',
-  );
+
+  try {
+    new FERAL_FUNCTION('async function* AsyncGeneratorFunctionInstance() {}')();
+    repairFunction(
+      'AsyncGeneratorFunction',
+      '%InertAsyncGeneratorFunction%',
+      '(async function*(){})',
+    );
+  } catch {
+    //
+  }
```

Comment:
add comment why this is here

Reply:
done in https://github.com/endojs/endo/pull/2334/commits/f2bc8f273c1185144710b6c0a6e51a5c94e402b4